jlm/rvsdg/gamma.hpp

```diff
@@ -286,6 +286,29 @@ class gamma_node : public jlm::rvsdg::structural_node {
 	jlm::rvsdg::gamma_output *
 	add_exitvar(const std::vector<jlm::rvsdg::output*> & values);
 
+  /**
```

Comment:
This seems to be a generic functionality, so why isn't it part of the structural node instead?

Reply:
The function is specific to the semantics of the gamma node. I guess the comment could be more specific saying that it removes all outputs with the corresponding results. The function is a little bit the odd one out in the PR, but I did not want to remove it again as I will need it anyway later.